src/view/info_frame.py

```diff
@@ -183,10 +183,14 @@ def start_keyboard_listener(self):
             on_press=self.__on_press,
             on_release=self.__on_release,
         )
-        self.listener.start()
+        if sys.platform != 'darwin':
```

Comment:
Perhaps we should wrap this in a Try/Except, where we can let the user know why exactly it's not working.

It's a shame that this guy's PR has been ignored for so long. Not too hopeful for a swift fix.

In the meantime -- I assume this means there's no keyboard shortcut for abording a bot. On Mac, is there a killswitch shortcut that automatically closes a process? Maybe holding `Cmd + Q`? If so, that should be added to the exception message to give the user a heads-up on how they can abort the bot if things get out of control.

Or maybe, a better solution would be to find a package that is specific to Mac and use that instead...

Thoughts?

Reply:
To stop the bot I just stop the debug session or close the app window or switch to another workspace (with touchpad) where it doesn't have any marker to click on. Although inconvenient, it's not something that caused me any problems yet.

> Or maybe, a better solution would be to find a package that is specific to Mac and use that instead...

I thought about doing this re-write when `keyboard` package was used as it wasn't officially supported MacOS, however pynput was suggested at the time that was said to not have such problems.

I'm not sure it's worth rewriting this feature entirely just for MacOS system because it doesn't hurt so much to keep it as it is now (at least for me and at least yet). I mean, this kinda ducktapes the issue until a proper solution is found just to avoid people asking about how to make this project work on Mac.

The purpose of the `if` is just to keep the app going with a warning that keyboard stuff may not work as expected and a clear condition that it would not work on `darwin` (I will add a print statement). A generic `Try/Except` without re-raising exception would give the same result  but with a less explicit code, wouldn't it?

Reply:
> A generic Try/Except without re-raising exception would give the same result but with a less explicit code, wouldn't it?

Yeah, same result basically. I wasn't sure if the exception message had any useful info to help explain why it wasn't working, but yea as long as it's clear to the user that keybinds won't work, should be alright for now.

Maybe even suggest those workarounds you mentioned in the print message in case the person ran the project in _run mode_ instead of _debug mode_. Sometimes people write scripts that totally hijack their mouse and leave them stranded lol.

Reply:
By the way, it reminded me it's not an exception, it's a segfault, try/catch won't handle it.